src/components/FeatureFlagModal/FeatureFlagModal.tsx

```diff
@@ -202,6 +203,18 @@ export default function FeatureFlagModal() {
           <X size={24} />
         </CloseButton>
       </Header>
+      <FeatureFlagOption
```

Comment:
maybe a group called "features"

Reply:
I don't think we should group them if they are unrelated